src/store/setting.ts

```diff
@@ -157,11 +133,30 @@ export const settingStore = createPartialStore<SettingStoreTypes>({
         });
       }
 
-      commit("SET_ENABLE_MULTI_ENGINE", {
-        enableMultiEngine: await window.electron.getSetting(
-          "enableMultiEngine"
-        ),
-      });
+      const rootMiscSettingKeys = [
+        "editorFont",
+        "showTextLineNumber",
+        "showAddAudioItemButton",
+        "splitTextWhenPaste",
+        "splitterPosition",
+        "enableMultiEngine",
+      ] as const;
+
+      // rootMiscSettingKeysに値を足し忘れていたときに型エラーを出す検出用コード
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const _: IsEqual<
+        keyof RootMiscSetting,
+        typeof rootMiscSettingKeys[number]
+      > = true;
+
+      for (const key of rootMiscSettingKeys) {
+        commit("SET_ROOT_MISC_SETTING", {
+          // NOTE: Vuexの型処理でUnionが解かれてしまうのを迂回している
```

Comment:
```suggestion
          // Vuexの型処理でUnionが解かれてしまうのを迂回している
```

NOTE: は他のところで使ってない（多分）ので消してもいいかも？

Reply:
まあ確かに消しても良さそうに思いました！